book/04-git-server/sections/protocols.asc

```diff
@@ -53,7 +53,7 @@ If you and a co-worker are working on the same project and they want you to chec
 
 ===== The Cons
 
-The cons of this method are that shared access is generally more difficult to set up and reach from multiple locations than basic network access.
+The cons of this method are that shared access is generally more difficult to set up and reach from multiple locations than basic network access, and it is easier to damage the repository.
```

Comment:
Yes, this can be added as a small paragraph just before the HTTP protocols next chapter. I think inserting it above would break a serie of linked paragraphs.

It can take the form below:
> Finally, this protocol does not protect the repository against mistaken damages, since it is accessible as any standard directory to anyone having access to it.

Maybe the word "also" in the meanwhile paragraph *"It's __also__ important to mention"* will have to be removed to increase global readability when we insert the new paragraph. What do you think?